### DISCUSSION TITLE
Device is disabled.TH05_v1.5.

### DISCUSSION THREAD

**Discussion by misuvas** [pvvx/THB2#59] [Q&A]
Hi, please help with TH05_v1.5. I try to connect (in  chrome) but always show me: 
`12:00:19 AM: Not connected
12:00:27 AM: Searching for devices
12:00:58 AM: Waiting for connection to demo
12:01:06 AM: NetworkError: Connection Error: Connection attempt failed.
12:01:06 AM: Reconnect 0 / 5
12:01:06 AM: Waiting for connection to demo
12:01:13 AM: Wrong device selected!
12:01:13 AM: Device is disabled.`
What i do wrong? In tuya app phone works even in iot.dev.tuya.platform show device offline . i try to connect when was removed from tuya app. Still nothing.

**Comment by pvvx** [comment]
To start working the thermometer with “PHY62x2BTHome.html” you need to [reflash it using a USB-UART adapter](https://github.com/pvvx/THB2#firmware).
